index.html

```diff
@@ -9,5 +9,86 @@
   <header>
     <h1>US States Quiz</h1>
   </header>
-  <body></body>
+  <body>
+    <section class="quiz-question-block">
+      <h2 class="quiz-question">In which US State was Kool-Aid invented?</h2>
+      <button
+        aria-label="Click to Show Answer"
+        class="button-style"
```

Comment:
A more descriptive class name could be maybe "answer-button", or?

Reply:
Yes! Thank you!